server/src/main/java/org/apache/iotdb/db/query/dataset/DeviceIterateDataSet.java

```diff
@@ -175,15 +175,15 @@ protected boolean hasNextWithoutConstraint() throws IOException {
             groupByPlan.setSlidingStep(slidingStep);
             groupByPlan.setUnit(unit);
             groupByPlan.setDeduplicatedPaths(executePaths);
-            groupByPlan.setDeduplicatedDataTypes(dataTypes);
+            groupByPlan.setDeduplicatedDataTypes(tsDataTypes);
```

Comment:
Why changed this?

Reply:
I think DeviceIterateDataSet needs to do the job of removing duplicate paths